src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -11,10 +12,12 @@
 
 class RegistrationServiceImplTest {
     private static RegistrationService registrationService;
+    private static StorageDaoImpl storageDao;
```

Comment:
```suggestion
    private static StorageDao storageDao;
```
let's use abstraction instead of implementation